### DISCUSSION TITLE
zh/docs/basic/server-installation

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2064] [Docs comments]
# zh/docs/basic/server-installation

安装和配置 Casdoor 服务器

https://casdoor.org/zh/docs/basic/server-installation

**Comment by bytew021** [comment]
可以提供二进制单文件部署吗

**Reply by hsluoyz** [reply]
可以Docker部署：https://casdoor.org/zh/docs/basic/try-with-docker

**Reply by bytew021** [reply]
可以提供二进制单文件部署吗

**Reply by bytew021** [reply]
二进制单文件部署 可以让部署跟简单， 更有利推广

**Reply by hsluoyz** [reply]
@bytew021 can you create a GitHub issue in English at: https://github.com/casdoor/casdoor/issues ?

**Reply by explorer-exe-bilibili** [reply]
其实你编译出来的文件就可以单文件运行了

**Reply by explorer-exe-bilibili** [reply]
好吧是浏览器缓存.....
